packages/schematics/angular/application/other-files/app.component.spec.ts.template

```diff
@@ -3,16 +3,16 @@ import { RouterTestingModule } from '@angular/router/testing';<% } %>
 import { AppComponent } from './app.component';
 
 describe('AppComponent', () => {
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({<% if (routing) { %>
+  beforeEach(async () => {
+    return TestBed.configureTestingModule({<% if (routing) { %>
```

Comment:
Is there any point in using `async` if you are not `await`ing the promise?
I would expect either `async () => { await TestBed... }` or `() => { return TestBed... }`.

Reply:
i'd probably go with `await` to make it more explicit that it is an asynchronous operation and also slightly simplify future user additions (either before or after the call).

Reply:
I think async shouldn't only be used when awaiting, returning the promise also showcases asynchronous behavior.